test/parse_test.js

```diff
@@ -604,6 +604,19 @@ it('should apply `x-parser-spec-parsed` extension', async function() {
   await expect(parsedSpec.json()[String(xParserSpecParsed)]).to.equal(true);
 });
 
+it('should throw schema-parser-not-registered error on unrecognized schema', async function() {
```

Comment:
please use `checkErrorWrapper` as in other tests in case you need to asses different fields from the error object